wire-ios/Wire-iOS/Sources/UserInterface/Settings/Debug Report/SettingsDebugReportViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import MessageUI
+import UIKit
+import WireDataModel
+import WireDesign
+
+class SettingsDebugReportViewController: UIViewController {
+
+    // MARK: - Types
+
+    private typealias Strings = L10n.Localizable.Self.Settings
+
+    // MARK: - Properties
+
+    private let viewModel: SettingsDebugReportViewModelProtocol
+
+    // MARK: - Views
+
+    private let infoLabel: UILabel = {
+        let label = UILabel()
+        label.text = Strings.TechnicalReport.info
+        label.textColor = SemanticColors.Label.textDefault
+        label.numberOfLines = 0
+        label.textAlignment = .left
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private lazy var sendReportButton: UIButton = {
+        return createButton(
+            title: Strings.TechnicalReport.sendReport.capitalized,
+            action: #selector(didTapSendReport)
+        )
+    }()
+
+    private lazy var shareReportButton: UIButton = {
+        return createButton(
+            title: Strings.TechnicalReport.shareReport.capitalized,
+            action: #selector(didTapShareReport)
+        )
+    }()
+
+    // MARK: - Life cycle
+
+    init(viewModel: SettingsDebugReportViewModelProtocol) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = SemanticColors.View.backgroundDefault
+        navigationItem.setDynamicFontLabel(title: Strings.TechnicalReportSection.title.capitalized)
```

Comment:
Regarding Navigation Bar Titles isn't recommend it to make the text grow. I know that we already do in many places but I'm planning to fix that. Instead there's a feature called [Large Content Viewer](https://www.blog.kevin-hirsch.com/dynamic-type-large-content-viewer/)

Here's how that would work in this case:

```swift
func setupNavigationBarTitle() {
        // Set the navigation item title
        navigationItem.title = "Your Title"

        // Set the navigation bar title color and font
        let titleTextAttributes: [NSAttributedString.Key: Any] = [
            .foregroundColor: SemanticColors.Label.textDefault,
            .font: UIFont.font(for: .h3)
        ]
        navigationController?.navigationBar.titleTextAttributes = titleTextAttributes

        // Ensure title supports LargeContentViewer
        navigationItem.titleView?.isAccessibilityElement = true
        navigationItem.titleView?.accessibilityTraits = .header
        navigationItem.titleView?.accessibilityLabel = navigationItem.title

        if let titleView = navigationItem.titleView {
            titleView.showsLargeContentViewer = true
            titleView.largeContentTitle = navigationItem.title
        }
    }
```

It's also recommend it to setup the navigationBar in viewWillAppear. More info can be found [viewWIllAppear docs](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621510-viewwillappear)